sdk/webpubsub/Microsoft.Azure.WebPubSub.Common/src/Shared/WebPubSubStatusCode.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.WebPubSub.Common
+{
+    /// <summary>
+    /// The general enum to represent response status.
+    /// DO set to Success for backward capability.
+    /// Has mapping relationship to <see cref="WebPubSubErrorCode"/>.
+    /// </summary>
+    internal enum WebPubSubStatusCode
+    {
+        /// <summary>
+        /// Start from -1 to make values map <see cref="WebPubSubErrorCode"/>.
+        /// </summary>
+        Success = -1,
```

Comment:
Success should be the default one since we allow not set "code" for success responses